pages/placeholders.py

```diff
@@ -62,6 +62,11 @@ def parse_placeholder(parser, token):
                 params['widget'] = remaining[1]
             if bit == 'on':
                 params['page'] = remaining[1]
+            if bit == 'in':
+                quotes = ('"', "'")
+                if remaining[1].startswith(quotes) and remaining[1].endswith(quotes):
+                    remaining[1] = remaining[1][1:-1]
```

Comment:
You should use unescape_string_literal here

https://docs.djangoproject.com/en/1.9/_modules/django/utils/text/